src/sentry/models/tombstone.py

```diff
@@ -30,8 +30,8 @@ class Meta:
 
     __relocation_scope__ = RelocationScope.Excluded
 
-    table_name = models.CharField(max_length=48, null=False)
-    object_identifier = BoundedBigIntegerField(null=False)
+    table_name = models.CharField(max_length=48, null=False, db_index=True)
+    object_identifier = BoundedBigIntegerField(null=False, db_index=True)
```

Comment:
Should we have a composite index? Postgres will only pick one index for the filtering operation, a composite index would help narrow the number of rows when both conditions are used.